src/brutus-api/brutus_api/tasks.py

```diff
@@ -0,0 +1,41 @@
+from redis import Redis
+from rq import get_current_job
+
+from brutus_api import app
+from brutus_api import nlp
+import json
+
+from watson_developer_cloud import AuthorizationV1
+
+
+classifierName = "brutus-api"
```

Comment:
yes it probably should be. Do you this this should go in app.py or be an environment variable like the username and password?